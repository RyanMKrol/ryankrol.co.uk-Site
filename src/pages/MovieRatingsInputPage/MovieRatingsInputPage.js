import React from 'react';

import { MovieRatingsForm } from '../../components';

import styles from './MovieRatingsInputPage.module.css';

/**
 * Page introducing the site
 *
 * @returns {string} JSX Content
 */
function MovieRatingsInputPage() {
  return (
    <div className={styles.container}>
      <h1 className="light">Rate a movie!</h1>
      <MovieRatingsForm />
    </div>
  );
}

export default MovieRatingsInputPage;
