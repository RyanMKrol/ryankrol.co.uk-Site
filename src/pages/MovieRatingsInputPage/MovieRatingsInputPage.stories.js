import React from 'react';

import MovieRatingsInputPage from './MovieRatingsInputPage';

export default {
  title: 'Pages/MovieRatingsInputPage',
};

export const Default = () => <MovieRatingsInputPage />;
