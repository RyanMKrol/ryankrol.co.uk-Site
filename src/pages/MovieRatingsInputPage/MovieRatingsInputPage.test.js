import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import MovieRatingsInputPage from './MovieRatingsInputPage';

it('renders MovieRatingsInputPage correctly', () => {
  const header = shallow(<MovieRatingsInputPage />);
  expect(toJson(header)).toMatchSnapshot();
});
