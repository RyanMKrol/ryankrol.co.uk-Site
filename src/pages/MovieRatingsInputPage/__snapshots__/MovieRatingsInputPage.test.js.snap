// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MovieRatingsInputPage correctly 1`] = `
<div
  className="container"
>
  <h1
    className="light"
  >
    Rate a movie!
  </h1>
  <MovieRatingsForm />
</div>
`;
